RadarSDK/RadarLogger.m

```diff
@@ -70,7 +70,9 @@ - (void)logWithLevel:(RadarLogLevel)level type:(RadarLogType)type message:(NSStr
 
                 os_log(OS_LOG_DEFAULT, "%@", log);
 
-                [[RadarDelegateHolder sharedInstance] didLogMessage:log];
+                dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
Why does this need to use the main thread?

Reply:
I guess this is the typical pattern we use throughout the codebase callbacks

Reply:
Here we are calling the dispatcher, we should use the main thread as the lister on the other end may decide to use it to update the UI. e.g. have a super user log console that print out logs as they occur.

Reply:
Looking back at this, I'm thinking we shouldn't have this on the main thread. Unlikely a production user is gonna update UI based on the logs, and if there's lots of logging happening this could be contributing to main thread bloat

Reply:
Sounds right to me. But this is technically a breaking change so I'm adding a note in migration guide for completeness.